recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -444,6 +469,7 @@ def package_info(self):
                 self.cpp_info.components["opentelemetry_exporter_otlp_grpc_client"].requires.extend([
                     "grpc::grpc++",
                     "opentelemetry_proto",
+                    "opentelemetry_proto_grpc",
```

Comment:
`opentelemetry-cpp::proto_grpc` was added in Version "1.9.1" but you added this dependency in case Version >= 1.7  instead of >= 1.9 as I suggest you:

https://github.com/conan-io/conan-center-index/blob/c10b5e1353a4741681c0dad4d56303a091184674/recipes/opentelemetry-cpp/all/conanfile.py#L459-L462

Reply:
Erm derp